spec/services/imo_api_service_spec.rb

```diff
@@ -40,7 +40,7 @@
       before { Flipper.enable(:imo_messaging) }
 
       it "creates an ImoAuthorization on 200 success" do
-        stub_request(:post, request_url).with(headers: request_headers).to_return(status: 200, body: success_body)
+        # stub_request(:post, request_url).with(headers: request_headers).to_return(status: 200, body: success_body)
```

Comment:
I'm guessing this was to test the safety measure? We can revert this before shipping.